packages/connector-jsdom/tests/invalid-url.ts

```diff
@@ -26,5 +36,11 @@ test(`[${name}] Load an invalid url throws an error`, async (t) => {
 
     t.context.connector = connector;
 
-    await t.throws(connector.collect(new URL('https://localhome')));
+    t.plan(1);
```

Comment:
Another place for avoiding `plan` using `t.throwsAsync` per https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md#promises-with-a-catch-block

Reply:
connector.collect doesn't throw an error:
`Promise rejected with exception that is not an error:`

Reply:
Got it.